pkg/manifests/manifests.go

```diff
@@ -0,0 +1,27 @@
+package manifests
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"sigs.k8s.io/yaml"
+)
+
+// Read a Yaml file and unmarshal the contents
+func GetFileData(fileName string, output interface{}) {
+
+	path := filepath.Join("./manifests", fileName)
+
+	contents, err := os.ReadFile(path)
+	if err != nil {
+		err = fmt.Errorf("Error reading file %s: %w", path, err)
+	} else if err = yaml.Unmarshal(contents, output); err != nil {
+		err = fmt.Errorf("Error unmarshalling contents of %s: %w", path, err)
+	}
+
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
```

Comment:
Looks like a good idea to refactor in a common method, even though I'm not sure that all the errors may deserve an `os.Exit(1)`. In any case, I guess it's a decision that shold be take up into the stack, ie this method should just return the `error` and delegate its handling to someone else

Reply:
Changed to return error from GetFileData() and log error in calling function. In this case we want to exit if manifest file can't be read or is invalid, to avoid difficult to debug problems later.